backend/urls.py

```diff
@@ -15,8 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.views.generic import RedirectView
 
 urlpatterns = [
     path("api/", include("backend.api.urls")),
     path("admin/", admin.site.urls),
+    path("", RedirectView.as_view(url="/api/", permanent=True)),
```

Comment:
I have a had time to understand the purpose here.

Reply:
nope. Can you explain ?

Reply:
When an empty path is requested, for example `http://localhost:8000/`, it redirects automatically to `http://localhost:8000/api/`

Reply:
I don't see why we need that ^^. Since it's an api, we should let it as is right. Why would someone write `http://localhost:PORT/` ? If it happens, it should return an error, no ?

Reply:
This "someone" could be me 😂 But you're right, it won't have any usage in prod.